Dockerfile.dev

```diff
@@ -1,15 +1,15 @@
-# Small linux image with iter8 binary
+# Small Linux image with Iter8 binary
```

Comment:
Could we please remove the `Dockerfile.dev` file altogether? We can (later, in a separate PR), create an *example* of how to create your own Dockerfile with Iter8 in it (for local development purposes).

We really don't want to maintain this. For e.g., this is starting to diverge now from the main Dockerfile.

Reply:
Okay, sure

Reply:
Done